css/index.css

```diff
@@ -46,25 +49,36 @@ h1 {
   line-height: 1.4;
   
   */
-  
+  .content-row p{
+    font-size: 16px;
+    font-family: roboto;
+    padding: 10px;
+    line-height: 1.4;  
+  }
   
   /*======= Create a selector for ALL images in the site to have the following styles: 
   
     border-radius: 10px;
   
   */
+  .content-row img{
+    border-radius: 10px;
+  }
   
   .content-row {
     display: flex;
+    border-radius: 10px;
   }
   
   .content-row div {
     /*======= create a margin propery.  Set the value to 20px */
+    margin: 20 px;
   }
```

Comment:
This is incorrect syntax. The px needs to be right next to the '20'. 
Run your code in the browser and use your dev tools to detect code that is incorrectly formatted like this.
Also, VScode will sometimes have a little red dot in the left margin of the code where syntax errors are made.